tests/ui/rfc-2091-track-caller/error-with-main.stderr

```diff
@@ -2,9 +2,7 @@ error: `main` function is not allowed to be `#[track_caller]`
   --> $DIR/error-with-main.rs:1:1
    |
 LL | #[track_caller]
-   | ^^^^^^^^^^^^^^^
-LL | fn main() {
-   | --------- `main` function is not allowed to be `#[track_caller]`
```

Comment:
What happened to this span? I think it's still useful to point this out.

Reply:
I did not know how to work it in, and it did not look so important. But it's back now, thanks for [the help](https://github.com/rust-lang/rust/pull/108337#discussion_r1115066086).